packages/story-editor/src/components/checklist/checks/videoOptimization.js

```diff
@@ -52,24 +52,13 @@ const OptimizeButton = styled(Button)`
 `;
 
 export function videoElementsNotOptimized(element = {}) {
```

Comment:
Refactored this logic to not worry about state at all. All we care about with this function now, is it `isOptimized` and is the correct size.

Reply:
IIRC, this function used to do `if (isTranscoding) { return true; }`, so that videos which are currently being transcoded are still shown in the checklist. Is that still the case?

Reply:
It is in my testing.

Reply:
https://user-images.githubusercontent.com/237508/144456628-82f6c3c5-b29a-47e1-a4c0-687eaa50b11e.mov